src/main.ts

```diff
@@ -23,7 +23,7 @@ async function main() {
 
     // Initialize action inputs
     let endpoint: IAuthorizer = !!core.getInput('publish-profile') ? null : await AuthorizerFactory.getAuthorizer();
-    let actionParams: ActionParameters = ActionParameters.getActionParams(endpoint);
+    ActionParameters.getActionParams(endpoint);

```

Comment:
Is this required ?

Reply:
We need endpoint first time we initialize the object. Doing it in main instead of any other utility makes more sense